apps/astarte_realm_management/lib/astarte_realm_management/queries.ex

```diff
@@ -1280,31 +1023,22 @@ defmodule Astarte.RealmManagement.Queries do
     end
   end
 
-  def retrieve_trigger_uuid(client, trigger_name, format \\ :string) do
-    trigger_uuid_query_statement =
-      "SELECT value FROM kv_store WHERE group='triggers-by-name' AND key=:trigger_name;"
+  def retrieve_trigger_uuid(realm_name, trigger_name, format \\ :string) do
```

Comment:
Same comment as the previous one: a centralized logic to interact with the KvStore abstracting the parsing/casting logic might be useful.